include/igraph_eulerian.h

```diff
@@ -36,8 +36,8 @@
 
 __BEGIN_DECLS
 
-DECLDIR int igraph_is_eulerian(igraph_t *graph);
-DECLDIR int igraph_eulerian_paths(igraph_t *graph, igraph_vector_t *res);
+DECLDIR igraph_integer_t igraph_is_eulerian(igraph_t *graph, igraph_bool_t *has_path, igraph_bool_t *has_cycle);
+DECLDIR igraph_integer_t igraph_eulerian_paths(igraph_t *graph, igraph_vector_t *res);
```

Comment:
The return type should remain `int`. Error codes are always `int` in igraph. (You can take a look at some existing igraph functions for reference.)